webpack.config.prod.js

```diff
@@ -20,7 +20,13 @@ module.exports = {
       compress: {
         warnings: false
       }
-    })
+    }),
+    new webpack.optimize.OccurrenceOrderPlugin(),
```

Comment:
Well, mainly we don't care about file size in `dev`. I think the occurrence order one at least mangles the names of functions as well - although I'm not sure about that.